locations/spiders/enterprise.py

```diff
@@ -6,7 +6,6 @@
 
 class EnterpriseSpider(scrapy.Spider):
     name = "enterprise"
-    item_attributes = { 'brand': "Enterprise Rent-A-Car", 'brand_wikidata': "Q17085454" }
```

Comment:
Did you mean to remove this?

Reply:
Yes, because the response includes the brand, so I added it to the properties.

Reply:
Ah, ok. I don't see `brand` in the list of properties below. Can you please add it there?

Reply:
Woops! I'll it in here shortly.

Reply:
Turns out I was mistaken, I must have gotten the responses from sister companies Alamo and National confused. I added back in the above line.